scm/driver/gitea/user_test.go

```diff
@@ -20,10 +20,10 @@ func TestUserFind(t *testing.T) {
 	defer gock.Off()
 
 	gock.New("https://try.gitea.io").
-		Get("/api/v1/user").
+		Get("/api/v1/User").
 		Reply(200).
 		Type("application/json").
-		File("testdata/user.json")
+		File("testdata/User.json")
```

Comment:
changed from `user.json` to `User.json` intentional for due to find / replace?

Reply:
updated.